docs/contributing/contribution_guides/development.md

```diff
@@ -0,0 +1,89 @@
+2. Opening a new issue or feature to add. **Please make sure you have searched through existing issues before creating new ones.**
+
+*Regardless of what you choose, please post in the  [#dev-chat](https://discord.com/channels/1020123559063990373/1049495067846524939) channel of the Discord before you start development in order to confirm that the issue or feature is aligned with the current direction of the project. We value our contributors time and effort and want to ensure that no one’s time is being misspent.*
+
+## Best Practices: 
+* Keep your pull requests small. Smaller pull requests are more likely to be accepted and merged
+* Comments! Commenting your code helps reviwers easily understand your contribution
+* Use Python and Typescript’s typing systems, and consider using an editor with [LSP](https://microsoft.github.io/language-server-protocol/) support to streamline development
+* Make all communications public. This ensure knowledge is shared with the whole community
+
+## **How do I make a contribution?**
+
+Never made an open source contribution before? Wondering how contributions work in our project? Here's a quick rundown!
+
+Before starting these steps, ensure you have your local environment [configured for development](../LOCAL_DEVELOPMENT.md).
+
+1.  Find a [good first issue](https://github.com/invoke-ai/InvokeAI/contribute) that you are interested in addressing or a feature that you would like to add. Then, reach out to our team in the [#dev-chat](https://discord.com/channels/1020123559063990373/1049495067846524939) channel of the Discord to ensure you are  setup for success. 
+2. Fork the [InvokeAI](https://github.com/invoke-ai/InvokeAI) repository to your GitHub profile. This means that you will have a copy of the repository under **your-GitHub-username/InvokeAI**.
+3. Clone the repository to your local machine using:
+
+```bash
+git clone https://github.com/your-GitHub-username/InvokeAI.git
+```
+
+1. Create a new branch for your fix using:
+
+```bash
+git checkout -b branch-name-here
+```
+
+1. Make the appropriate changes for the issue you are trying to address or the feature that you want to add.
+2. Add the file contents of the changed files to the "snapshot" git uses to manage the state of the project, also known as the index:
+
+```bash
+git add insert-paths-of-changed-files-here
+```
+
+1. Store the contents of the index with a descriptive message.
+
+```bash
+git commit -m "Insert a short message of the changes made here"
+```
+
+1. Push the changes to the remote repository using
+
+```markdown
+git push origin branch-name-here
+```
+
+1. Submit a pull request to the **development** branch of the InvokeAI repository.
```

Comment:
We don't currently have an active development branch. We tried this and decided it was easiest to make PRs against main.

Reply:
Updated to be PRs against main

Reply:
Updated to be PRs against main